lib/Chunk.js

```diff
@@ -43,13 +75,23 @@ const getModulesSize = set => {
 	return count;
 };
 
+/**
+ * @class
+ * @name Chunk
+ * @description A Chunk is a unit of encapsulation for Modules.
+ * Chunks are "rendered" into bundles that get emitted when the build completes.
+ */
 class Chunk {
+	/**
+	 * @param {string=} name of chunk being created, is optional (for subclasses)
+	 */
 	constructor(name) {
 		this.id = null;
```

Comment:
every member should have a `@type` comment (and optionally `@private`)

Reply:
done